pycbc/events/coinc.py

```diff
@@ -850,21 +850,20 @@ def pick_best_coinc(cls, coinc_results):
     @classmethod
     def from_cli(cls, args, num_templates, analysis_chunk, ifos):
         from . import stat
+
         # Allow None inputs
-        if args.statistic_files is None:
-            args.statistic_files = []
-        if args.statistic_keywords is None:
-            args.statistic_keywords = []
+        stat_files = args.statistic_files or []
+        stat_keywords = args.statistic_keywords or []
```

Comment:
it seems the parse_statistic_keywords_opt function requires a list, so does it crash if given None?

Reply:
That is exactly the point of this code, making sure that `parse_statistic_keyword_opt()` receives an empty list instead of `None`.

Reply:
See previous comments. There doesn't seem to be a case where args.statistic_keywords could evaluate to False although a valid option had been given, so OK.  (Default values of these two options seems to be [] anyway.)